locales/en.default.schema.json

```
@@ -1214,6 +1214,15 @@
         "name": "Image with text"
       }
     },
+    "main-account": {
```

```suggestion
    "account": {
```

Not sure if we need `main` in there. There is only one type of account I believe.